ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java

```diff
@@ -86,9 +86,9 @@ public PreparedStatement<?> prepare(
 
       return PreparedStatement.of(stmt.getUnMaskedStatementText(), root);
     } catch (final ParseFailedException e) {
-      if (!e.getSqlStatement().isEmpty()) {
-        throw e;
-      }
+      //      if (!e.getSqlStatement().isEmpty()) {
+      //        throw e;
+      //      }
```

Comment:
We should keep this actually. Cf https://github.com/confluentinc/ksql/pull/9842/commits/1289c837ce421300ec4bc96f2285def3495dcdcc

Reply:
Ah I see